drivers/spi/spi_ll_stm32.c

```diff
@@ -480,6 +480,22 @@ static int spi_stm32_configure(const struct device *dev,
 		return -ENOTSUP;
 	}
 
+	/* configure the frame format Motorola (default) or TI */
+	if ((config->operation & SPI_FRAME_FORMAT_TI) == SPI_FRAME_FORMAT_TI) {
+#ifdef LL_SPI_PROTOCOL_TI
+		LL_SPI_SetStandard(spi, LL_SPI_PROTOCOL_TI);
+#else
+		LOG_ERR("Frame Format TI not supported");
+		/* on stm32F1 or some stm32L1 without SPI_CR2_FRF */
+		return -ENOTSUP;
+#endif
+	} else {
+#if defined(LL_SPI_PROTOCOL_MOTOROLA) && defined(SPI_CR2_FRF)
+		/* on stm32F1 or stm32L1 (cat1,2) do not raise an error, it is default state */
+		LL_SPI_SetStandard(spi, LL_SPI_PROTOCOL_MOTOROLA);
+#endif
```

Comment:
This has to be changed otherwise will result on empty `else` which will be rejected by static analysis tools